testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksInsertCompatibility.java

```diff
@@ -484,4 +498,162 @@ public void testWritesToTableWithGeneratedColumnFails()
             dropDeltaTableWithRetry("default." + tableName);
         }
     }
+
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS}, dataProviderClass = DataProviders.class, dataProvider = "trueFalse")
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testConcurrentInsertsReconciliation(boolean partitioned)
```

Comment:
We don't really have write reconciliation with Databricks though. Until we agree on something like the DynamoDB locking mechanism we won't.

Reply:
Indeed, we're not reconciling with Databricks here.
But we need somehow to test that `INSERTS` are working while other operations are performed concurrently.
Spark can do DML in a concurrent fashion, so it is temporarily (until we have concurrent reconciliation on other DML operations in Trino ) a good sparing partner for testing.